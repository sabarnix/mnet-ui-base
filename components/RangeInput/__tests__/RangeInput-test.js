"use strict";

var _react = _interopRequireDefault(require("react"));

var _reactTestRenderer = _interopRequireDefault(require("react-test-renderer"));

require("jest-styled-components");

var _MnetUIBase = require("../../MnetUIBase");

var _ = require("..");

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { "default": obj }; }

test('RangeInput renders', function () {
  var component = _reactTestRenderer["default"].create( /*#__PURE__*/_react["default"].createElement(_MnetUIBase.MnetUIBase, null, /*#__PURE__*/_react["default"].createElement(_.RangeInput, {
    value: "50"
  })));

  var tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});