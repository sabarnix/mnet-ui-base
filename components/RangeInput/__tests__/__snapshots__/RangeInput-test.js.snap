// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RangeInput renders 1`] = `
.c0 {
  font-size: 12px;
  line-height: 16px;
  box-sizing: border-box;
  -webkit-text-size-adjust: 100%;
  -ms-text-size-adjust: 100%;
  -moz-osx-font-smoothing: grayscale;
  -webkit-font-smoothing: antialiased;
}

.c1 {
  box-sizing: border-box;
  position: relative;
  -webkit-appearance: none;
  border-color: transparent;
  height: 16px;
  width: 100%;
  padding: 0px;
  cursor: pointer;
  background: transparent;
}

.c1::-moz-focus-inner {
  border: none;
}

.c1::-moz-focus-outer {
  border: none;
}

.c1::-webkit-slider-runnable-track {
  box-sizing: border-box;
  width: 100%;
  height: 4px;
  background: rgba(205,211,227,0.2);
}

.c1::-webkit-slider-thumb {
  margin-top: -6px;
  box-sizing: border-box;
  position: relative;
  border-radius: 16px;
  height: 16px;
  width: 16px;
  overflow: visible;
  background: #E15151;
  -webkit-appearance: none;
  cursor: pointer;
}

.c1::-webkit-slider-thumb:hover {
  box-shadow: 0px 0px 0px 2px #E15151;
}

.c1::-moz-range-track {
  box-sizing: border-box;
  width: 100%;
  height: 4px;
  background: rgba(205,211,227,0.2);
}

.c1::-moz-range-thumb {
  box-sizing: border-box;
  position: relative;
  border-radius: 16px;
  height: 16px;
  width: 16px;
  overflow: visible;
  background: #E15151;
  -webkit-appearance: none;
  cursor: pointer;
  margin-top: 0px;
  height: 16px;
  width: 16px;
}

.c1::-ms-thumb {
  box-sizing: border-box;
  position: relative;
  border-radius: 16px;
  height: 16px;
  width: 16px;
  overflow: visible;
  background: #E15151;
  -webkit-appearance: none;
  cursor: pointer;
  margin-top: 0px;
  height: 16px;
  width: 16px;
}

.c1:hover::-moz-range-thumb {
  box-shadow: 0px 0px 0px 2px #E15151;
}

.c1:hover::-ms-thumb {
  box-shadow: 0px 0px 0px 2px #E15151;
}

.c1::-ms-track {
  box-sizing: border-box;
  width: 100%;
  height: 4px;
  background: rgba(205,211,227,0.2);
  border-color: transparent;
  color: transparent;
}

.c1::-ms-fill-lower {
  background: rgba(205,211,227,0.2);
  border-color: transparent;
}

.c1::-ms-fill-upper {
  background: rgba(205,211,227,0.2);
  border-color: transparent;
}

<div
  className="c0"
>
  <input
    className="c1"
    onBlur={[Function]}
    onChange={[Function]}
    onFocus={[Function]}
    type="range"
    value="50"
  />
</div>
`;
